tests/test_swapped_models.py

```diff
@@ -0,0 +1,165 @@
+import importlib
+import os
+import subprocess
+
+from django.conf import settings
+from django.db import migrations
+from django.test import TestCase
+from django.test.utils import override_settings
+
+
+class TestSwappableModels(TestCase):
+    """
+    This test verifies that it is possible to install `django-scheduler`
+    and then swap its models.
+    """
+    settings_path = os.path.join(settings.BASE_DIR, 'tests', 'test_project', 'settings.py')
+    updated_settings_path = os.path.join(settings.BASE_DIR, 'tests', 'test_project', 'updated_settings.py.tpl')
+
+    def setUp(self):
+        super(TestSwappableModels, self).setUp()
+        # cleanup any possible leftovers before running
+        self._cleanup()
```

Comment:
Why do you need to call `_cleanup` both before and after? Shouldn't one be sufficient?

Reply:
if for some reason the test completely crashes, this makes sure the test "setup" isn't corrupted. I can easily remove, it is just a nice-to-have for the developers of this repo.